### DISCUSSION TITLE
More DAW features

### DISCUSSION THREAD

**Discussion by ghost** [tenacityteam/tenacity-legacy#544] [Ideas]
It would be lovely if Tenacity could have more DAW features. We already have a very strong base here, we just need a few more "details" :-)

1) an easy way to repeat a loop/sample. Something like move your cursor over the side of a loop that you want to extend, it turns into a circular arrow, which tells you that you can click and repeat the loop. Now, Tenacity would add multiple copies of the same loop automatically, for as far as you drag the loop.

2) Plugins per track. Since Tenacity is multi-track, it would be nice to manage plugins per track. That would be a huge step forward, a more DAW approach. 

3) MIDI love. Be able to add/delete/edit MIDI files. Something like the "piano roll". 

That's all we need to make music with Tenacity!

**Comment by poyribn** [comment]
þis sounds lovely.
I twice tried making music wiþ LMMS, and þe results weren't good because I couldn't use þe program very well.
If tenacity adds þese bare-minimum DAW features, I could make "actual" music: I don't need more DAW features þan þese since oþerwise þe program would be too complicated.
